pages/tools/fileconverter.vue

```diff
@@ -0,0 +1,407 @@
+<template>
+  <v-container>
+    <v-row class="flex-column">
+      <v-col>
+        <h1 class="text-h2 py-5" align="center">
+          File converter
+        </h1>
+        <v-col>
+          <v-row justify="space-around">
+            <v-col v-for="(item, i) in items" :key="i" cols="11" md="5">
+              <v-card
+                v-ripple
+                class="card"
+                nuxt
+                hover
+                elevation="5"
+                :href="item.href"
+                target="_blank"
+                contain
+              >
+                <v-row justify="center" align="center">
+                  <v-col cols="auto">
+                    <v-icon
+                      v-if="item.icon.match(/mdi-.*/)"
+                      size="128"
+                      class="justify-center"
+                    >
+                      {{ item.icon }}
+                    </v-icon>
+                    <v-img
+                      v-else
+                      :src="require('@/assets/logo.svg')"
+                      alt="Geode-solutions logo"
+                      width="50%"
+                      class="mx-auto"
+                      contain
+                    />
+                  </v-col>
+                </v-row>
+                <v-card-title
+                  primary-title
+                  class="justify-center text-h6"
+                  align="center"
+                >
+                  <br>
+                  {{ item.title }}
+                </v-card-title>
+                <v-card-text class="justify-center text-body-1">
+                  {{ item.text }}
+                </v-card-text>
+              </v-card>
+            </v-col>
+          </v-row>
+        </v-col>
+      </v-col>
+      <v-col v-if="!cloudRunning">
```

Comment:
This will be a perfect example to learn about slots. Remember this comment, we will look at them for the next tool.